design/MonitoringModeAPI.md

```diff
@@ -70,14 +70,17 @@ see [Update results](/design/UpdateResults.md)
 
 **NOTE:** The update to results of a particular experiment should follow Time Series Order to get valid recommendations.
```

Comment:
We will need to remove references to Time series order or chronological order as we are now supporting non-chronological order. Please update the documentation

Reply:
done